Tasks/DownloadPipelineArtifactV0/task.json

```diff
@@ -9,7 +9,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 151,
+        "Minor": 139,
```

Comment:
Why is this going down?

Reply:
This was changed by mistake by this PR : https://github.com/Microsoft/azure-pipelines-tasks/pull/10139/files, but we didn't change the agent plugin version, hence reverting.